graphdatascience/tests/integration/test_graph_ops.py

```diff
@@ -351,6 +367,14 @@ def test_graph_nodeProperties_stream_without_arrow(gds_without_arrow: GraphDataS
     y_values = result[result.nodeProperty == "y"]
     assert {e for e in y_values["propertyValue"]} == {2, 3, 4}
 
+    z_values = result[result.nodeProperty == "z"]
+    assert len(z_values) == 3
+    for e in z_values["propertyValue"]:
+        assert e in [[9], [42], [1337]]
```

Comment:
could be worth to write our own test-util function for this if we use it this often